src/main/java/duke/task/Storage.java

```diff
@@ -0,0 +1,116 @@
+package duke.task;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates a storage object that stores a list of tasks in a fixed path.
+ */
+public class Storage {
+
+    ArrayList<Task> al;
```

Comment:
Perhaps the variable name could be more explicit? Same issue for other initiated collection types as well.